commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleCoreTasks.scala

```diff
@@ -201,6 +201,26 @@ trait UnicycleCoreTasks {
     }
   }
 
+  class RotateToAngle(absoluteAngle: Angle, tolerance: Angle)
+                     (implicit drive: Drivetrain,
+                      hardware: DrivetrainHardware,
+                      props: Signal[DrivetrainProperties]) extends FiniteTask {
+    override def onStart(): Unit = {
+      val (controller, error) = turnPositionControl(absoluteAngle)
+      val checkedController = controller.withCheck { _ =>
+        if (error.get.abs < tolerance) {
+          finished()
+        }
+      }
+
+      drive.setController(lowerLevelVelocityControl(speedControl(checkedController)))
```

Comment:
Instead of writing a controller to convert a unicycle signal to a unicycle velocity, let's just add a constructor to unicycleVelocity that takes a unicycleSignal.